cmd/trivy/main.go

```diff
@@ -12,9 +13,8 @@ var (
 )
 
 func main() {
-	app := commands.NewApp(version)
-	err := app.Run(os.Args)
-	if err != nil {
+	app.Version = version
```

Comment:
Is there any reason why we cannot this version?
https://github.com/aquasecurity/trivy/blob/1b66b77f691f2249a63c284056afcad704c0e15f/pkg/commands/option/global.go#L16